Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -6,19 +6,10 @@ import Foundation
 import Snappy
 import UIKit
 
-// This is the bounding box of the button. The image is aligned to the top of the box, the text label to the bottom.
-private let ButtonSize = CGSize(width: 72, height: 56)
-
-// Color and height of the orange divider
-private let DividerColor: UIColor = UIColor(red: 255.0 / 255, green: 149.0 / 255, blue: 0.0 / 255, alpha: 1)
-private let DividerHeight: CGFloat = 4.0
-
-// Font name and size used for the button label
-private let LabelFontName: String = "FiraSans-Light"
-private let LabelFontSize: CGFloat = 13.0
-
-private let BackgroundColor = UIColor(red: 57.0 / 255, green: 57.0 / 255, blue: 57.0 / 255, alpha: 1)
-private let TransitionDuration = 0.25
+private let BackgroundColor = UIColor(red: 45.0 / 255, green: 52.0 / 255, blue: 66.0 / 255, alpha: 1)
+private let NormalIconColor = UIColor.lightGrayColor()
+private let SelectedIconColor = UIColor(red: 62.0 / 255, green: 136.0 / 255, blue: 255.0 / 255, alpha: 1)
+private let ButtonToolbarHeight = 48
```

Comment:
Can't for a couple of reasons:
  1. Our Target Membership is a bit broken right now. Because of PanelTests, HomePanelViewController is in the ClientTests target, but BVC is not. Adding BVC to ClientTests means we'd need to add a slew of other classes to ClientTests.
  1. The [mockups](https://mozilla.invisionapp.com/share/EG26NE528#/screens) show that the button bar height is a little smaller than the URL bar height.